encrypted-media-respec.html

```diff
@@ -1240,7 +1259,31 @@
                         </dd>
                         <dt><a def-id="persistent-usage-record-session"></a></dt>
                         <dd>
-                          <p>Let <var>requested license type</var> be a non-persistable license that will persist a <var>record of key usage</var>.</p>
+                          <ol>
+                            <li>
+                              <p>Initialize this object's <var>record of key usage</var> as follows.</p>
+                              <ul>
+                                <li>
+                                  <p>
+                                    Set the list of <a def-id="key-id">key IDs</a> <a href="#known-key">known</a> to the session to the empty list.
+                                  </p>
+                                </li>
+                                <li>
+                                  <p>
+                                    Set the <var>first decrypt time</var> to <code>null</code>.
```

Comment:
I think this is a slightly different usage from `expiration` where we decided in `NaN` in issue #58. In that case, this is a real value for `expiration` with the meaning that there is no expiration. In this case we are using `null` as a temporary placeholder to indicate that the first / latest decryption has not happened yet. It's not much of a difference but it's not obvious that there's any particular reason to align on `NaN`.